### PR TITLE
ReadMe Tweaks

### PR DESCRIPTION
**Pull request by jthelin**
- Fix some end-of-line character problems in README.md

- Smooth out some formatting wrinkles in README.md

### CODE REVIEW

**Comment by jthelin** [comment]
Whenever I pull master on a Windows machine, the line endings in README.md file get auto-fixed so seems to need this one-time update. 

This PR also fixes some formatting kinks in README.md so it is readable in text mode as well as .md mode.

File: README.md [hunk 1]
```diff
@@ -1,122 +1,163 @@
...
-        clean.cmd
-        ```   
-
-## Running Samples
-### Prerequisites
-DataFrame TextFile API uses `spark-csv` package to load data from CSV file. Latest [commons-csv-*.jar](http://commons.apache.org/proper/commons-csv/download_csv.cgi) and [spark-csv*.jar (Scala version:2.10)](http://spark-packages.org/package/databricks/spark-csv) should be downloaded manually.
-
-The following environment variables should be set properly:
-* ```JAVA_HOME```  
-* ```SCALA_HOME```
```
Comment (danielli90) [inline README.md]: SCALA_HOME disappeared when rendered by github - https://github.com/jthelin/SparkCLR/blob/README/README.md

It seems replace {```} with {```} fixed - e.g. - https://github.com/danielli90/SparkCLR/blob/mydev/README.md

Comment (danielli90) [inline README.md]: added proper escape - I meant "replace {\`\`\`} with {\`}" (replace three backticks with one backtick}

Comment (jthelin) [inline README.md]: Good catch @danielli90 -- updated to only use three-backticks for code blocks and single-backtick for inline code.

**Pull request merged**